jobs/mysql/templates/drain_user_setup.sql.erb

```diff
@@ -0,0 +1,5 @@
+<%
+  node_ip = spec.ip
+  hosts = (node_ip.split(".")[0..-2] + ["%"]).join('.')
+%>
+GRANT USAGE ON mysql.* TO 'drain'@'<%= hosts %>' IDENTIFIED BY '<%= p('cf_mysql.mysql.drain.db_password') %>'
```

Comment:
It would be best to find some Ruby syntax to SQL-escape those values that are injected here.